src/plugins/raven/selectors/getCollaborationNodes.js

```diff
@@ -0,0 +1,8 @@
+import getNode from '@triniti/cms/plugins/ncr/selectors/getNode';
+
+export default (state) => {
+  const fakeArticle = 'acme:article:fake-article';
```

Comment:
@nanetteCodes not javascript constants, raven constants :]

so that here and in pollCollaborationsFlow are sharing the same constants constantly

Reply:
updated this